packages/container/lib/container.js

```diff
@@ -350,7 +474,7 @@ function instantiate(container, fullName) {
 
     if (typeof factory.extend === 'function') {
```

Comment:
does this refer to the `factory.class` or our new factory object, if it is our new factory object this may no longer make sense?